examples/source/1.components/amp-script/index.html

```diff
@@ -0,0 +1,132 @@
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <!--
+
+        ## Setup
+
+        First, you need to import the `amp-script` extension.
+      -->
+      <script async custom-element="amp-script" src="https://cdn.ampproject.org/v0/amp-script-0.1.js"></script>
+      <style amp-custom>
+        .amp-script-sample {
+          padding: 8px;
+        }
+        .amp-script-sample button {
+          background-color: #005AF0;
+          font-size: 16px;
+          font-weight: 700;
+          color: white;
+          border: none;
+          padding: 8px;
+          margin: 8px 0;
+        }
+      </style>
+
+      <!-- 
+        For inline scripts you need to generate a CSP header. Read more about this in the [component documentation](content/amp.dev/documentation/components/reference/amp-script.md)).
+      -->
+      <meta name="amp-script-src" content="sha384-tHJJ6ba4Oo1ocFdMe9v9zz9YrhMyy5Lvi4L_Fp-7sU-AlEHqctUbulcGepcA61YP sha384-nNFaDRiLzgQEgiC5kP28pgiJVfNLVuw-nP3VBV-e2s3fOh0grENnhllLfygAuU_M sha384-u7NPnrcs7p4vsbGLhlYHsId_iDJbcOWxmBd9bhVuPoA_gM_he4vyK6GsuvFvr2ym">
+  </head>
+  <body>
+      <!-- 
+        ## Basic usage
+
+        An amp-script element can load a JavaScript file from a URL:
+      -->
+      <amp-script layout="container" src="<% hosts.platform %>/hello-world.js" class="amp-script-sample" >
+        <button>Click!</button>
+      </amp-script>
+
+      <!-- 
+        This is the script in [hello-world.js](hello-world.js):
+
+        ```js
+const button = document.querySelector('button');
+button.addEventListener('click', () => {
+  const h1 = document.createElement('h1');
+  h1.textContent = 'Hello World!';
+  document.body.appendChild(h1);
+});```
+
+        ## Inline Scripts
+
+        You can also declare scripts inline and reference them by `id`.
+      -->
+      <amp-script layout="container" script="hello-world" class="amp-script-sample">
+        <button>Click!</button>
+      </amp-script>
+
+      <!-- This is the inlined script. Note that the type needs to be `type=text/plain` and the `target=amp-script`: -->
+      <script id="hello-world" type="text/plain" target="amp-script">
+    const button = document.querySelector('button');
+    button.addEventListener('click', () => {
+      const h1 = document.createElement('h1');
+      h1.textContent = 'Hello World 2!';
+      document.body.appendChild(h1);
+    });
+  </script>
+  <!--
+      ## Using the fetch API
+
+      `amp-script` supports using the fetch API. To be able to update the page on load, we need to use the `fixed-height` layout instead of the `container` layout.
```

Comment:
"...and the height has to be 300px or less"

Reply:
done